src/Compilers/Core/Portable/CommandLine/ErrorLogger.cs

```diff
@@ -66,19 +68,15 @@ internal void LogDiagnostic(Diagnostic diagnostic, CultureInfo culture)
                 message = "<None>";
             }
 
-            string description = diagnostic.Descriptor.Description.ToString(culture);
```

Comment:
Was this broken previously? i.e., that we only emitted a message in the event of a non-null/empty diagnostic.Decriptor.Description?